doc/changelog/04-tactics/19703-lia-print-used-hyps.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  The :flag:`Info Micromega` (unset by default) makes :tacn:`lia`,
+  :tacn:`lra`, :tacn:`nia` and :tacn:`nra` to print the names of
```

Comment:
```suggestion
  :tacn:`lra`, :tacn:`nia` and :tacn:`nra` print the names of
```

FWIW, it would be great to be able to print similar information such as the list of theorems used in a completed proof or by an Ltac/Ltac2 tactic.  Or to get a list of proofs that depend on a group of theorems or modules.  But far outside the scope of this PR.